Tests/App/config/config_test.yaml

```diff
@@ -21,3 +21,9 @@ doctrine:
                 type: annotation
                 dir: '%kernel.root_dir%/../../Entity'
                 prefix: ClaviculaNox\PendingActionsBundle\Entity
+
+services:
+    fake.service:
+         class: ClaviculaNox\PendingActionsBundle\Tests\FakeBundle\Classes\FakeService
+         arguments:
+             []
```

Comment:
If arguments are empty, the whole `arguments:` statement can be removed. And a newline is needed at the end of the file